src/js/global-profile.language.js

```diff
@@ -37,6 +37,57 @@
 					case 'GUE':
 						$rootScope.locale = 'en-US';
 						break;
+					case 'RWA':
+						$rootScope.locale = 'fr';
+						break;
+					case 'BEN':
+						$rootScope.locale = 'fr';
+						break;
+					case 'BFA':
+						$rootScope.locale = 'fr';
+						break;
+					case 'BDI':
+						$rootScope.locale = 'fr';
+						break;
+					case 'CMR':
+						$rootScope.locale = 'fr';
+						break;
+					case 'CAF':
+						$rootScope.locale = 'fr';
+						break;
+					case 'TCD':
+						$rootScope.locale = 'fr';
+						break;
+					case 'COG':
+						$rootScope.locale = 'fr';
+						break;
+					case 'GAB':
+						$rootScope.locale = 'fr';
+						break;
+					case 'GIN':
+						$rootScope.locale = 'fr';
+						break;
+					case 'CIV':
+						$rootScope.locale = 'fr';
+						break;
+					case 'MLI':
+						$rootScope.locale = 'fr';
+						break;
+					case 'NER':
+						$rootScope.locale = 'fr';
+						break;
+					case 'SEN':
+						$rootScope.locale = 'fr';
+						break;
+					case 'TGO':
+						$rootScope.locale = 'fr';
+						break;
+					case 'ETH':
+						$rootScope.locale = 'am';
+						break;
+					case 'IDE':
+						$rootScope.locale = 'id';
+						break;
```

Comment:
Sorry, I wanted PR 5 & 6, but _not_ PR 7. IDE and the 'id' language should not be here (avoid GP-41), but the 'am' language should be.